pkg/proxy/nftables/proxier.go

```diff
@@ -189,6 +190,8 @@ type Proxier struct {
 	// serviceCIDRs is a comma separated list of ServiceCIDRs belonging to the IPFamily
 	// which proxier is operating on, can be directly consumed by knftables.
 	serviceCIDRs string
+
+	logger klog.Logger
```

Comment:
nftables's `syncProxyRules` uses `context.TODO()`, so you should keep the whole context so you can fix that

Reply:
ok, if we're not going to store the context then this gets a little more complicated so I guess that could be fixed separately later